websites/api-docs-smoke-test/package.json

```diff
@@ -9,6 +9,7 @@
     "serve": "npx gatsby serve --prefix-paths -o",
     "clean": "npx gatsby clean",
     "clean:develop": "yarn clean && yarn develop",
+    "create-docs-release-note": "PATH=$(npm bin):$PATH create-docs-release-note",
```

Comment:
Note: I added this to the package.json scripts of each microsite so that the script can be run in the "npm scripts" window in vs code. Otherwise it's not visible

Reply:
Hmm this looks wrong. What are you trying to do here? 🤔